expyfun/analyze/_viz.py

```diff
@@ -368,7 +368,7 @@ def barplot(h, axis=-1, ylim=None, err_bars=None, lines=False,
             # raise apex a bit extra if prev bracket label centered on bar
             prev_label_pos = brk_c[_ix - 1] if _ix else -1
             label_bar_ix = np.where(np.isclose(bar_centers, prev_label_pos))[0]
-            if label_bar_ix in _brk:
+            if any(np.array_equal(label_bar_ix, x) for x in _brk):
```

Comment:
it wasn't an error, it was a warning. It's in current `master` for me. I am running `numpy` 1.9 and Ubuntu 15.04 (so maybe newer Python 2.7 version, too), so maybe it's from one of those.